PatientRegistrationApp/src/components/SinglePatientContactInfo.vue

```diff
@@ -3,15 +3,17 @@
     <v-row>
       <v-col cols="12" sm="6" md="3">
         <v-text-field
-          :rules="['Required']"
+        required
+		:rules="[v => !!v || 'Phone number is required']"
           label="Primary Phone Number"
           v-model="patientPhoneNumber"
         ></v-text-field>
       </v-col>
 
       <v-col class="d-flex" cols="6" sm="2">
         <v-select
-          :rules="['Required']"
+        required
+		:rules="[v => !!v || 'Phone type is required']"
```

Comment:
Phone type is marked as "required" here, but not checked in the verifyFormContents() function.  According to Balsamiq, it looks like phone type is not required.